htdocs/comm/propal/card.php

```diff
@@ -2008,7 +2008,8 @@
 			$sday = date("d", $tmpdte);
 			print $form->selectDate($syear."-".$smonth."-".$sday, 'date_livraison', '', '', '', "addprop");
 		} else {
-			print $form->selectDate(-1, 'date_livraison', '', '', '', "addprop", 1, 1);
+			$tmp_date_start = GETPOST('date_start') ? : -1;
+			print $form->selectDate($tmp_date_start, 'date_livraison', '', '', '', "addprop", 1, 1);
```

Comment:
Why using name date_start and not date_livraison (in english date_delivery) ?

Reply:
you are totally right.
I did this because we use it as a "start date" but it makes no sense in code. Force push is done